config/elasticsearch.yml

```diff
@@ -1,6 +1,6 @@
 default: &DEFAULT
   hosts:
-    - <%= ENV['ES_HOST'] || 'localhost:9200' %>
+    - <%= ENV['ES_HOSTS'] || 'localhost:9200' %>
```

Comment:
Made this plural to match [`search-gov`](https://github.com/GSA/search-gov/blob/e285f8a54b44694cefd781f01c0be084726dec98/config/secrets.yml.dev#L61).